global_helpers/test_panther_greynoise_helpers.py

```diff
@@ -0,0 +1,271 @@
+import pytest
```

Comment:
@rmarathay - we don't know how this will affect the Detections backend. For the tests, we'll have to follow [the example set here](https://github.com/panther-labs/panther-analysis/blob/master/global_helpers/panther_oss_helpers.py#L435-L488) until we're able to build in something to support what you're trying to do. Notably, you'll have to use the style of testing that example uses with the `assert`s